_author_instructions/01_policies.md

```diff
@@ -204,9 +204,15 @@ After this time, a revised manuscript may be handled as a new submission.
 
-## After acceptance
-
-After acceptance, the authors should perform a careful readthrough of the manuscript for errors, as no additional typesetting or proofing will be done after that point.  Specific tasks that need to be carried out to add issue numbers are listed [the editor instructions](https://livecomsjournal.github.io/policies/editor_instructions/#following-acceptance). After being published as an ASAP article, the authors will have one final chance to fix minor typos and inaccuracies before the article is published in the next regularly scheduled issue.  Beyond that, any typos will remain in the LiveCOMS version until the next peer-reviewed version.  Changes can of course be made at any time in the authors’ maintained version on GitHub, which is linked from the article itself.  
+## Upon acceptance
+
+ Make sure the metadata is complete in the submission (including institution, ORCID, abstract). 
+- Have the managing editor assign a DOI to the article (correct metadata is needed for this).
+- Do a final check for issues with grammar, word choice, spelling, formatting (including issues with the LaTeX template, placement of figures, etc.) and typos, as normally would be done at the proofs stage. Please [reference the appropriate part of the author instructions](https://livecomsjournal.github.io/authors/policies/#writing-style-and-editing) for what to look for. 
+- Enable the LiveCoMS footer by including the 'pubversion' class option in the document preamble, fill in the '\datereceived', '\dateaccepted', and '\pubDOI' fields (which the managing editor will provide after the Metadata is verified to be correct. You should make the PDF accessible on the GitHub repository as `release/LiveCoMS_Article_VX.pdf`, where 'X' is the current article verrsion. The related header image should be at least 1200 pixels wide and in `.jpg` format, and should be available on the GitHub repository as `release/header_VX.jpg`, where again 'X' is the current article version, and notify the editor when this has been done.
+- Provide the editor who managed the article a "lede", a 30 word summary of the article that will accompany the article in social media posts.
+- When assigned to an issue (you will be notified by the  the '\pubvolume', '\pubyear', and '\articlenum' fields, recompile the PDF, and then provide to a managing editor to update the PDF.
```

Comment:
Some words left out here, also paren issues.
Should we say "When assigned to a JOURNAL issue, ..." to avoid confusion with a github issue?

> recompile the PDF, and then provide to a managing editor to update the PDF.

Can we omit the last 'to update the PDF' or change to 'for publication as the ASAP version'?

Reply:
OK, fixed this up according to the comments.